src/style/app.module.css

```
@@ -4188,3 +4194,1521 @@ button[data-testid='createPostBtn'] {
+  font-size: 16px;
+  cursor: pointer;
+}
+
+/* YearlyEventCalender.tsx */
+
+.calendar__weekdays {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  background-color: var(--grey-dark);
+  height: 60px;
+  role: 'row';
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)